.travis.yml

```diff
@@ -84,3 +70,24 @@ script:
 after_script:
     - echo $TRAVIS_COMMIT_RANGE
     - echo $TRAVIS_COMMIT_LOG
+
+jobs:
+  include:
+    - stage: CHECK_DOC
+      env:
+        - HOST=arm-linux-gnueabihf PACKAGES="g++-arm-linux-gnueabihf python3-pip shellcheck" DEP_OPTS="NO_QT=1" CHECK_DOC=1 GOAL="install" BITCOIN_CONFIG="--enable-glibc-back-compat --enable-reduce-exports"
```

Comment:
Please keep the actual arm cross build intact. The only thing you'd need to move here is the pip and shellcheck package

Also, would be nice if the check doc run before the tests, not the other way round.

Reply:
`HOST=arm-linux-gnueabihf` will be executed twice in the `check_doc` and `test` then?